letsencrypt/acme/messages.py

```diff
@@ -74,35 +37,58 @@ def get_msg_cls(cls, jobj):
         try:
             msg_cls = cls.TYPES[msg_type]
```

Comment:
Nitpick: Realize this code didn't change but it might be easier to just return cls.TYPES[msg_type] rather than saving the variable and returning after the except.  Not a problem though.